src/plugins/Freereader/Freereader.java

```diff
@@ -75,12 +81,71 @@ public void terminate()
 		Logger.normal(this, pluginName + " terminated.");
 	}
 	
-	public String getString(String key) {
-		// TODO return a translated string
-		return key;
+	/**
+	 * This code is only used by FredPluginL10n...
```

Comment:
Simply specify than this call should not be used outside of l10n logic.
- WoT : no comment - https://github.com/freenet/plugin-WebOfTrust/blob/master/src/plugins/WebOfTrust/WebOfTrust.java#L3832
- FlogHelper : same comment - https://github.com/freenet/plugin-FlogHelper/blob/master/src/plugins/floghelper/FlogHelper.java#L435
- Freetalk : same comment - https://github.com/freenet/plugin-Freetalk/blob/master/src/plugins/Freetalk/Freetalk.java#L783

... remove or not remove, change comment or leaving like this ?